### ISSUE TITLE
Warning when searching for contributions - 

### ISSUE THREAD

**Issue by themak1985** [eileenmcnaughton/nz.co.fuzion.omnipaymultiprocessor#146]
Any idea why we would get this warning when finding contributions?

`Warning: Declaration of CRM_Core_Payment_PaymentExtended::getPaymentDescription($params, $length = 24) should be compatible with CRM_Core_Payment::getPaymentDescription($params = Array, $length = 24) in require_once() (line 34 of /home/columbiaccs/www/www/sites/default/files/civicrm/ext/nz.co.fuzion.omnipaymultiprocessor/CRM/Core/Payment/PaymentExtended.php).`

Using 5.21.1 D7, and 3.8

**Comment by eileenmcnaughton** [comment]
@themak1985 yeah -it needs this patch - https://github.com/eileenmcnaughton/nz.co.fuzion.omnipaymultiprocessor/commit/c8e05ed7639926adf557a8d42e384a2dfc9cd46a (in Omnipay 3.9)
